dh/sidh/internal/p434/arith_amd64.s

```diff
@@ -29,27 +29,36 @@
 //   |-128-| x |--- 256 ---| = |------ 384 ------|
 // Assuming the first digit multiplication was already performed.
 #define MULX128x256(I1, M1, T1, T2, T3, T4, T5)    \
-    MULXQ   M1+ 8(SB), T4, T2   \
+    MOVQ    M1+ 8(SB), AX       \
+    MULXQ   AX, T4, T2          \
```

Comment:
Yes it's unnecessary, but it is included due to bug in the compiler. (see go/issues/58735). So this is a workaround.

In short, what is happening here is that MULXQ cannot reference to the M1 location because is a global variable.

Reply:
Ah, makes sense!